<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="p1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>48</y>
     <width>859</width>
     <height>206</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12롯데마트드림Bold</family>
     <pointsize>47</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,140);
font: 47pt &quot;12롯데마트드림Bold&quot;;
gridline-color: rgb(0, 0, 0);
border:4px solid black;
border-color:rgb(209,254,225);
color:black;</string>
   </property>
   <property name="lineWidth">
    <number>-2</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot; style = &quot;vertical-align:middle&quot;&gt;&lt;span style=&quot; font-size:120pt; &quot;&gt;RVM&lt;/span&gt;&lt;span style=&quot; font-size:24pt; color:#000000;&quot;&gt;(일사분란착착착)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/img/bg.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="p2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>289</y>
     <width>535</width>
     <height>269</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12롯데마트드림Bold</family>
     <pointsize>143</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/newPrefix/img/pic4.png);
gridline-color: rgb(0, 0, 0);
border:4px solid black;
border-color:rgb(209,254,225);
color:black;</string>
   </property>
   <property name="lineWidth">
    <number>-2</number>
   </property>
   <property name="text">
    <string>
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="p2_1">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>318</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12롯데마트드림Bold</family>
     <pointsize>52</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>-2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="p2_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>433</y>
     <width>171</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>12롯데마트드림Bold</family>
     <pointsize>52</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>SizeVerCursor</cursorShape>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="lineWidth">
    <number>-2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="p3">
   <property name="geometry">
    <rect>
     <x>671</x>
     <y>289</y>
     <width>267</width>
     <height>267</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>12롯데마트드림Bold</family>
     <pointsize>83</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,140);
font: 83pt &quot;12롯데마트드림Bold&quot;;
gridline-color: rgb(0, 0, 0);
border:4px solid black;
border-color:rgb(209,254,225);
color:black;</string>
   </property>
   <property name="text">
    <string>시작</string>
   </property>
  </widget>
  <zorder>widget</zorder>
  <zorder>p1</zorder>
  <zorder>p2</zorder>
  <zorder>p2_1</zorder>
  <zorder>p2_2</zorder>
  <zorder>p3</zorder>
 </widget>
 <resources>
  <include location="bg.qrc"/>
 </resources>
 <connections/>
</ui>
